std_msgs/Header header
string traj_file
string ply_file
string scan_folder
float64 map_res
string raw_output_folder
float64[3] gravity
